### PR TITLE
Cherries now juice into cherry juice

### PR DESCRIPTION
**Pull request by SomeguyManperson**
cherry juice was set as grind result not juice result so it would NEVER SHOW UP making it COMPLETELY NONEXISTENT :)))
:cl:  
bugfix: cherry juice is now attainable again
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Important stuff ngl

File: code/modules/hydroponics/grown/berries.dm [hunk 1]
```diff
@@ -134,7 +134,7 @@
 	filling_color = "#FF0000"
 	bitesize_mod = 2
 	foodtype = FRUIT
-	grind_results = list(/datum/reagent/consumable/cherryjelly = 0)
+	juice_results = list(/datum/reagent/consumable/cherryjelly = 0)
```
Comment (Hopekz) [inline code/modules/hydroponics/grown/berries.dm:137]:
```suggestion
	juice_results = list(/datum/reagent/consumable/cherryjelly = 1)
```

File: code/modules/hydroponics/grown/berries.dm [hunk 2]
```diff
@@ -158,7 +158,7 @@
 	filling_color = "#6495ED"
 	bitesize_mod = 2
 	foodtype = FRUIT
-	grind_results = list(/datum/reagent/consumable/bluecherryjelly = 0)
+	juice_results = list(/datum/reagent/consumable/bluecherryjelly = 0)
```
Comment (Hopekz) [inline code/modules/hydroponics/grown/berries.dm:161]:
```suggestion
	juice_results = list(/datum/reagent/consumable/bluecherryjelly = 1)
```

**Review by Hopekz — CHANGES_REQUESTED** [review]
I don't think these will work if these are set to 0 as it might be getting 0 of the thing back after juice. 
Might be the whole reason it didn't exist in the first place after a grind from what I can tell since grinding vs juicing seems to just be flavortext + a different result if they have both grind and juice options.

**Comment by SomeguyManperson** [comment]
```
/obj/item/reagent_containers/food/snacks/grown/on_juice()
	var/nutriment = reagents.get_reagent_amount(/datum/reagent/consumable/nutriment)
	if(juice_results&&juice_results.len)
		for(var/i in 1 to juice_results.len)
			juice_results[juice_results[i]] = nutriment
		reagents.del_reagent(/datum/reagent/consumable/nutriment)
		reagents.del_reagent(/datum/reagent/consumable/nutriment/vitamin)
```

**Comment by Hopekz** [comment]
> ```
> /obj/item/reagent_containers/food/snacks/grown/on_juice()
> 	var/nutriment = reagents.get_reagent_amount(/datum/reagent/consumable/nutriment)
> 	if(juice_results&&juice_results.len)
> 		for(var/i in 1 to juice_results.len)
> 			juice_results[juice_results[i]] = nutriment
> 		reagents.del_reagent(/datum/reagent/consumable/nutriment)
> 		reagents.del_reagent(/datum/reagent/consumable/nutriment/vitamin)
> ```

clever code

**Review by fluffe9911 — APPROVED** [review]
hip

**Pull request merged**